src/components/ui/CustomInput/CustomInput.tsx

```diff
@@ -0,0 +1,142 @@
+import { Typography, Box } from '@mui/material';
+import React, { ChangeEvent, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface ICustomInputProps {
+  label: string;
+  value: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  disabled?: boolean;
+  id: string;
+  placeholder: string;
+  error?: string;
+  style?: React.CSSProperties;
+  type: string;
+}
+
+const styles = {
+  mainBox: {
+    width: '100%',
+  },
+  label: {
+    color: '#404142',
```

Comment:
need create variables for colors, in the future it will be hard to change, this colors need move to ui-toolkit (soon I will create it)